assets/scss/checkout.scss

```diff
@@ -1,3 +1,85 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
+
+// Main content
+.ng-checkout-container {
```

Comment:
@bc-miko-ademagic are we happy to use it for this purpose or would you suggest something else?